bridgetown-website/netlify.toml

```diff
@@ -0,0 +1,40 @@
+[build]
+  command = "yarn deploy && ./bin/netlify.sh"
+  publish = "output"
+
+[build.processing]
+  skip_processing = false
+[build.processing.css]
+  bundle = false
+  minify = false
+[build.processing.js]
+  bundle = false
+  minify = false
+[build.processing.html]
+  pretty_urls = true
+[build.processing.images]
+  compress = false
```

Comment:
We could probably remove these build rules if you'd like, but I do kind of like being explicit as to what processing we're doing :)